packages/apps/graph/src/index.ts

```diff
@@ -1,23 +1,17 @@
+require('module-alias/register');
```

Comment:
Can this be done with `import`?
```suggestion
import('module-alias/register');
```

Reply:
It can, but the auto sort for imports will move it down. For now it's okay, but it will break if an import is sorted above it that uses module aliases.

Reply:
Ah good one. Yes we can ignore the auto-fix I think?
This is one of the reasons we used to have a custom order. Sometimes you need a order when importing like that.

What happens when you use `import 'module-alias/register'`?